implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -427,6 +428,7 @@ public int queueSize() {
 
     /**
      * @deprecated queue capacity is no longer available since 1.4.0
+     * @return queue capacity
```

Comment:
Same as the queueSize
```suggestion
     * @return constant {@literal -1}
```